src/backend/settings/__init__.py

```diff
@@ -352,3 +348,50 @@
 }
 
 REQUIRED_SETTINGS = ["FRONTEND_URL", "DOMAIN"]
+
+EMAIL_ADDRESS = os.getenv("EMAIL_ADDRESS", "noreply@ractf.co.uk")
```

Comment:
can these settings be moved to config pls

Reply:
Not easily, they need to be set before the first user tries to login. I support us moving a lot of settings out to config if we have a better out of box experience but currently the platform expects to be ready on boot. I think we need to a wider discussion about this, but I think it's beyond the scope of this PR.

Reply:
Consider a config value with a default value.

Reply:
I stand by my comment above. Moving this into config would be good, but this is a setting we need to be able to provision before the first login, and that's not something we have the capability to do right now.

Reply:
Consider a config value with the default value set to `os.getenv("EMAIL_ADDRESS", "noreply@...")`

Reply:
In that instance, how do we handle the user changing the value in config and locking themselves out? Currently, this is a platform-level setting rather than user-level.

Reply:
Make this something like `ctf@example.com` so we don't have people trying to send emails as ractf.co.uk

Reply:
Resolved in b43b521e707d44d34ba178b2d43675309bea49f9